plat/common/lcpu.c

```diff
@@ -48,7 +48,7 @@ void ukplat_lcpu_halt_irq(void)
 	ukplat_lcpu_disable_irq();
 }
 
-void ukplat_lcpu_halt_to(__snsec until)
+void ukplat_lcpu_halt_to(__nsec until)
```

Comment:
Can you update your working branch according to the latest `staging`, please? Otherwise, we will get a conflict here, with this PR, that has been already merged: https://github.com/unikraft/unikraft/pull/257/commits/210ff11d7bac31801359a57ce010639fd0f6eeff

Reply:
Done.